### PR TITLE
fix arrest script

### PR DESCRIPTION
**Pull request by wescottj98**

### CODE REVIEW

File: mission/functions/systems/actions/fn_action_arrest_player.sqf [hunk 1]
```diff
@@ -19,8 +19,8 @@
 	format ["<t color='#B700FF'>%1</t>", localize 'STR_vn_mf_arrest_player'],							// Title of the action
 	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_secure_ca.paa",	// Idle icon shown on screen
 	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_secure_ca.paa",	// Progress icon shown on screen
-	"(player getVariable ['vn_mf_db_player_group', 'MikeForce'] in ['MilitaryPolice', 'DacCong']) && player distance cursorTarget <= 5 && primaryWeapon player != "" &&{ vehicle player isEqualTo player && {cursorTarget isKindOf 'Man' && {alive cursorTarget}}}",	// Condition for the action to be shown
-	"player distance cursorTarget < 7",						// Condition for the action to progress
+	player getVariable ["isArrested", "false"] == "false" && "(player getVariable ['vn_mf_db_player_group', 'MikeForce'] in ['MilitaryPolice', 'DacCong']) && player distance cursorTarget <= 3 &&{ vehicle player isEqualTo player && {cursorTarget isKindOf 'Man' && {alive cursorTarget}}}",	// Condition for the action to be shown
```
Comment (dijksterhuis) [inline mission/functions/systems/actions/fn_action_arrest_player.sqf:22]: What are you trying to do with the `player getVariable ["isArrested", "false"] == "false" && ` part of this?

File: mission/functions/systems/actions/fn_action_release_from_arrest_player.sqf [hunk 2]
```diff
@@ -19,8 +19,8 @@
 	format ["<t color='#33FF00'>%1</t>", localize 'STR_vn_mf_release_from_arrest_player'],							// Title of the action
 	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_secure_ca.paa",	// Idle icon shown on screen
 	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_secure_ca.paa",	// Progress icon shown on screen
-	"player getVariable ["isArrested"] && player distance cursorTarget <= 5 && { vehicle player isEqualTo player && {cursorTarget isKindOf 'Man' && {alive cursorTarget && }}}",	// Condition for the action to be shown
-	"player distance cursorTarget < 7",						// Condition for the action to progress
+	"player getVariable ["isArrested", "false"] == "true" && player distance cursorTarget <= 3 && { vehicle player isEqualTo player && {cursorTarget isKindOf 'Man' && {alive cursorTarget && }}}",	// Condition for the action to be shown
```
Comment (dijksterhuis) [inline mission/functions/systems/actions/fn_action_release_from_arrest_player.sqf]:
```
["isArrested", "false"] == "true"
```

double quotes within double quotes

Comment (dijksterhuis) [inline mission/functions/systems/actions/fn_action_release_from_arrest_player.sqf]: extra `&&` at the end might be doing something weird

File: mission/para_player_loaded_client.sqf [hunk 3]
```diff
@@ -45,6 +45,7 @@ if (_side == east) then {
 
 private _playerSide = side _player;
 _player setVariable ["vn_mf_side", _playerSide, true];
+_player setVariable ["isArrested", "true", true];
```
Comment (dijksterhuis) [inline mission/para_player_loaded_client.sqf]: Shouldn't this be false to start?

**Comment by wescottj98** [comment]
@dijksterhuis Can you take a look at this and test it.

**Pull request merged**